packages/x-charts/src/ChartsXAxis/ChartsXAxis.tsx

```diff
@@ -193,10 +194,10 @@ function ChartsXAxis(inProps: ChartsXAxisProps) {
   });
 
   const domain = xScale.domain();
-  if (domain.length === 0 || domain[0] === domain[1]) {
-    // Skip axis rendering if
-    // - the data is empty (for band and point axis)
-    // - No data is associated to the axis (other scale types)
+  // Skip axis rendering if
+  // - the domain is Infinite (for band and point axis)
```

Comment:
Why woudl it be only for ordinal scale?
```suggestion
  // - the domain is Infinite
```

Reply:
I was under the impression this would only happen in those scale types, but seems false